### DISCUSSION TITLE
How to match lenght of FileEdit to FigureImage with FixedSize?

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#2828] [Q&A] [answered]
Sorry for bombarding you like this on a sunday but I just can´t get this to work.

What ever I try to do I can´t get the `<FileEdit>` size to match the  `<Figure>` size

This code (and loads of variations of it)

```xml
<Column ColumnSize="ColumnSize.Is4.OnDesktop">
  <Field ColumnSize="ColumnSize.Is4.OnDesktop">
      <Figure Size="FigureSize.Is256x256">
         <FigureImage Source="@Image" />
      </Figure>
  </Field>
  <Field ColumnSize="ColumnSize.Is4.OnDesktop">
      <FileEdit Filter="image/*" Changed="@OnPosterImageChanged" />
  </Field>
</Column>
```
comes out like this
![image](https://user-images.githubusercontent.com/2386572/130357995-462f0d74-785c-4718-bdcb-32ce481a11cb.png)

If I then add `Style="width:256px;"` it seams that FileEdit gets the width but **the control doesn´t draw in the space**? Is that correct?

![image](https://user-images.githubusercontent.com/2386572/130358676-955dfd69-8831-4d11-8456-42df20dfe3b1.png)

I have at least tried lots of stuff without luck so I hope its not just me :-)

**Comment by sturlath** [comment]
Ok just thought of it.. I probably must use the new [Flex util](https://blazorise.com/docs/helpers/utilities/). I'll give it a try!

**Reply by sturlath** [reply]
Sorry.. its not clear how I should do this (if possible with Flex). 

I have tried to wrap those 2 in div and added Flex.Grow without change but I have also tried lots of other things... nothing I do seems to change it to matching size. Its probably something dead simple but I can´t figure it out.

**Comment by stsrki** [comment, marked as answer]
I think it's wrong since you're mixing fixed image sizes with dynamic column sizes. And there is no way to match them.

This is the closest I can think of. But even then the image would need to be of the right size to stay in the element boundaries.

```cshtml
<Row>
    <Column ColumnSize="ColumnSize.Is4.OnDesktop">
        <Field Border="Border.Primary">
            <Figure>
                <Image Source="assets/images/empty/256x256.png" Fluid />
            </Figure>
        </Field>
        <Field Border="Border.Secondary">
            <FileEdit Filter="image/*" />
        </Field>
    </Column>
</Row>
```

PS. added borders as a visual cue

![image](https://user-images.githubusercontent.com/900302/130454556-409eae9d-4f8a-4c08-9bb1-7864f8f96eb0.png)

**Reply by David-Moreira** [reply]
Hello @sturlath,
If you don't have a dynamic stylying requirement. And really want those fixed width of 256px. This should work:
![image](https://user-images.githubusercontent.com/22283161/130508714-dc2dc7bb-a198-40ac-b783-57cc75e26a78.png)
![image](https://user-images.githubusercontent.com/22283161/130508497-1ee017aa-daf0-4394-889f-a48f12873311.png)
```
<Row>
    <Column ColumnSize="ColumnSize.Is4.OnDesktop">
        <Figure>
            <Image Source="assets/images/empty/256x256.png" Fluid />
        </Figure>
        <Div Style="width:256px">
            <FileEdit Filter="image/*" />
        </Div>
    </Column>
</Row>
```

**Reply by sturlath** [reply]
Thanks @David-Moreira that totally works for me! Thank you both for your help! You are doing fantastic work!

**Reply by stsrki** [reply]
Can you mark the comment as Answered?

**Reply by sturlath** [reply]
Sorry I did’t find the answer button for @David-Moreira answer but I see it now that I can accept yours.
